pxtlib/tilemap.ts

```diff
@@ -356,6 +376,54 @@ namespace pxt {
             for (const id of Object.keys(allJRes)) this.state.takenNames[id] = true;
         }
 
+        loadJres(jres: Map<JRes>, skipDuplicates = false) {
+            jres = inflateJRes(jres)
```

Comment:
need to set `this.extensionTilesets = []` as in loadPackage ?

Reply:
no, this adds to the existing project instead of overwriting it. The gallery won't get added twice